bin/lookup

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+(async () => {
+    const args = process.argv.slice(2)
+    const { IPData } = require("../lib/ipdata")
+    const ip = args[0] || undefined
+    const apiKey = args[1] || process.env.APIDATA_API_KEY || undefined
```

Comment:
Is `process.env.APIDATA_API_KEY` correct?

Reply:
Haha, no - I hadn't actually gone through with final checks until I felt out if the shape of the PR was something you were interested in, or thought should fundamentally change etc -